write-fonts/src/pens.rs

```diff
@@ -548,4 +556,260 @@ mod tests {
         assert_eq!(Some(Rect::new(50.0, 50.0, 350.0, 300.0)), pen.bounds());
         assert!(pen.bounds().unwrap().area() > bez.bounding_box().area());
     }
+
+    // Direct port of fonttools' reverseContourPen_test.py::test_reverse_pen, adapted to rust,
+    // excluding test cases that don't apply because we don't implement outputImpliedClosingLine=False.
+    // https://github.com/fonttools/fonttools/blob/85c80be/Tests/pens/reverseContourPen_test.py#L6-L467
+    #[rstest]
+    #[case::closed_last_line_not_on_move(
```

Comment:
Given our offset use I think we should add a test with only a move and a test with only move/close. Will add prior to merging.